src/functions_framework/typed_event.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 Google LLC
```

Comment:
SHould the file name here start with an underscore? It's an internal implementaiton only file right?